openpype/hosts/houdini/api/colorspace.py

```diff
@@ -54,3 +54,19 @@ def get_colorspace_data(self):
             )
         ]
         return colorspace_data
+
+
+def get_default_display_view_colorspace():
+    """Get default display view colorspace.
+
+    It's used for 'ociocolorspace' parm in OpneGL Node."""
+
+    data = get_color_management_preferences()
+    config_path = data.get("config")
+    display = data.get("display")
+    view = data.get("view")
+
+    default_view_space = get_display_view_colorspace_name(config_path,
+                                                          display,
+                                                          view)
+    return default_view_space
```

Comment:
I'd personally prefer it like this - but it's cosmetics mostly.
```suggestion
    prefs = get_color_management_preferences()
    return get_display_view_colorspace_name(
        config_path=prefs["config"],
        display=prefs["display"],
        view=prefs["view"]
    )
```

Note that at the very least you don't need `.get()` calls because the function always returns those three keys in the dict.

Reply:
same here and that's also the preference of `black` formatter